controller/scrape.go

```diff
@@ -58,6 +58,7 @@ func GetLitmusChaosMetrics(clientSet *clientV1alpha1.Clientset) error {
 		total += totalEngine
```

Comment:
Also received some feedback on the total: `total` seems to denote total executed too, while we use it to say total listed experiments in the engine. Would be better to change the latter to say "engine_experiment_list" or "expected_count" or some such. The actual passed+failed value is a "derived" metric that can be obtained by an appropriate query